bundles/org.openhab.binding.meteoalerte/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -184,9 +166,131 @@
 		</config-description>
 	</thing-type>
 
-	<channel-type id="alert-level">
+	<channel-type id="vent">
+		<item-type>Number</item-type>
+		<label>Etat Vent</label>
```

Comment:
Maybe you can remove "Etat" from all the channel labels ?

Reply:
Done